src/plugin/message/Manager/MessageManager.php

```diff
@@ -178,26 +178,17 @@ public function send(Message $message, $setAsSent = true, $sendMail = true)
         return $message;
     }
 
-    public function sendMessageToAbstractRoleSubject(
-        AbstractRoleSubject $subject,
+    /**
+     * @param AbstractRoleSubject[] $receivers
+     */
+    public function sendMessage(
         $content,
         $object,
+        array $receivers = null,
         $sender = null,
         $withMail = true
     ) {
-        $users = [];
-
-        if ($subject instanceof User) {
-            $users[] = $subject;
-        }
-
-        if ($subject instanceof Group) {
-            foreach ($subject->getUsers() as $user) {
-                $users[] = $user;
-            }
-        }
-
-        $message = $this->create($content, $object, $users, $sender);
+        $message = $this->create($content, $object, $receivers, $sender);
```

Comment:
Create method expects an array of `User`, but receive an array of `AbstractRoleSubject` which can also be `Group`. You must keep the previous logic on the `$subject` var and apply it to all the elements of the `$receivers` array.

Reply:
I reverted my change about the previous logic, thx